cypress/integration/plugins/reports-dashboards/01-create.spec.js

```diff
@@ -0,0 +1,249 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { visitReportingLandingPage } from "../../../utils/plugins/reports-dashboards/constants";
+import { BASE_PATH } from '../../../utils/constants';
+
+describe('Adding sample data', () => {
```

Comment:
Can you add an `after` block to remove these tests? Best practice is to remove any created resources in case this is running against some persistent cluster that may not have it, or in the case that another plugin's test suite is trying to load the same sample data and fails due to it already being created

Reply:
Also, you can add this in a `before` block so it doesn't show up as an actual test for Reporting since it's unrelated

Reply:
Sure I can add something to remove the data- but the sample data won't fail if it's already created, I've investigated that before

Reply:
+ when running locally it's much easier to have the end state match the start state

Reply:
Good point, forgot to account that all plugins are testing on the same cluster here